src/lib.rs

```diff
@@ -63,13 +64,41 @@ pub mod pylib {
     }
 
     #[pyfunction]
-    pub fn refresh_logger() {
+    pub fn test_logging() {
```

Comment:
it might be cleaner to have this return the max log level instead as `get_max_log_level()`